core/src/test/java/nl/nn/adapterframework/http/rest/OpenApiTest.java

```diff
@@ -18,17 +19,126 @@
 
 	@Test
 	@IsolatedThread
-	public void simpleEndpointTest() throws Exception {
+	public void simpleEndpointGetTest() throws Exception {
 		ApiServiceDispatcher dispatcher = ApiServiceDispatcher.getInstance();
 		assertEquals("there are still registered patterns! Threading issue?", 0, dispatcher.getPatternClients().size());
 
-		new AdapterBuilder("myAdapterName", "description4simple-get").setListener("users", "get").setValidator("simple.xsd", null, "user").build(true);
+		new AdapterBuilder("myAdapterName", "description4simple-get").setListener("users", "get", null).setValidator("simple.xsd", null, "user", null).addExit("200").addExit("500").build(true);
 
 		assertEquals("more then 1 registered pattern found!", 1, dispatcher.getPatternClients().size());
 		String result = callOpenApi();
 
 		String expected = TestFileUtils.getTestFile("/OpenApi/simple.json");
 		TestAssertions.assertEqualsIgnoreCRLF(expected, result);
+		dispatcher.getPatternClients().clear();
+	}
+
+	@Test
+	@IsolatedThread
+	public void simpleEndpointPostTest() throws Exception {
+		ApiServiceDispatcher dispatcher = ApiServiceDispatcher.getInstance();
+		assertEquals("there are still registered patterns! Threading issue?", 0, dispatcher.getPatternClients().size());
+
+		new AdapterBuilder("myAdapterName", "description4simple-get").setListener("simpleEndpointPostTest", "post", null).setValidator("simple.xsd", null, "user", null).addExit("200").build(true);
```

Comment:
In this case (i.e. when building a configuration) it is more clear to have a separate line for each method